TAO/orbsvcs/IFR_Service/IFR_Service.mpc

```diff
@@ -3,6 +3,7 @@
 // necessary to add minimum_corba to 'avoids' here.
 project(IFR_Service) : orbsvcsexe, install, ifrservice, ifr_client, imr_client, svc_utils {
   exename = tao_ifr_service
+  install = $(ACE_ROOT)/bin
```

Comment:
This changes the location of the tao_ifr_service which can break test scripts and user setups, for installing it as part of the rpm it is not necessary to change the location of where it is build, see for example tao-cosnaming in the spec file

Reply:
I think I understand the issue you mentioned and adjusted my updates. I returned the IFR_Service.mpc to its original state and simply have the rpmbuild spec install from the original build location.